sheets/1/exercise-3.tex

```diff
@@ -36,22 +36,28 @@
+                  \item Kommandos beginnen häufig mit \texttt{set}, wenn sie dazu dienen direkt eine Variable/Attribut zu verändern. Komplexere in der Regeln nicht.
+              \end{itemize}
           \end{Infobox}
     \item \textbf{[1]} Ein Befehl endet immer mit einem ; und muss in einem Codeblock stehen.
     \item \textbf{[2]} Das erzeugen eines Objekts wird mit dem Schlüsselwort new eingeführt. Darauffolgt der Name der Klasse und (). Manchmal werden in die () noch Argumente geschreiben, mit Dingen die das Objekt von Anfang an braucht.
-          \begin{lstlisting}[title=\textbf{Kommando Syntax},firstnumber=5]
+          \begin{lstlisting}[title=\textbf{Konstruktor Syntax},firstnumber=5]
         new <Typ Name>();
     \end{lstlisting}
     \item \textbf{[2]} Wenn wir ein Objekt länger als nur einmal brauchen müssen wir uns dieses Merken. Dazu verwenden wir eine Variable. Um eine zu erstellen. Schreiben wir zuerst die Klasse zu der das Objekt gehört. Gefolgt von dem frei wälbaren Namen den die Variable haben soll. Mit einem = lassen wir sie dann auf ein Objekt zeigen, indem wir entweder das new Objekt() direkt dahinter schreiben oder eine andere
           Variable die bereits einen Wert hat.
-
-          \begin{Infobox}[Attribute]
-              Wenn man eine Variable in den Codeblock einer Operation schreibt, dann kann man sie nur in der Operation verwenden.
-              Wenn man sie aber außerhalb der Operation, direkt in die Klasse schreiben, wird sie nicht Variable sondern Attribut genannt und kann von allen Operation die zu der Klasse gehören verwendet werden.\\
-              Das ist außerdem eine Ausnahme bei der ein Befehl nicht in dem Codeblock einer Operation steht.
-          \end{Infobox}
-    \item \textbf{[2]} Es wurde ja bereits erwähnt, dass Klassen Operationen haben. Außerdem, dass sie "alle" Befehle beinhalten. Wenn man diese Aufrufen möchte: schreibt man den namen der Variable die auf das Objekt zeigt oder auch direkt einfach den Initializer. Dann einen . gefolgt von dem Namen der Operation und (). Manche geben aber auch ein Objekt zurück, dadurch kann man gleich einen Operator davon Aufrufen oder man speichert sich es, mit einer variable zwischen. Außerdem haben sie manchmal Parameter, wie beim Initatiliesieren.
+          \begin{lstlisting}[title=\textbf{Variablen Syntax}]
+    <Typ>
+          \end{lstlisting}
+    \item \textbf{[2]} Es wurde ja bereits erwähnt, dass Klassen Operationen haben. Außerdem, dass sie \glqq alle\grqq\ Befehle beinhalten. Wenn man diese Aufrufen möchte: schreibt man den namen der Variable die auf das Objekt zeigt oder auch direkt einfach den Initializer. Dann einen . gefolgt von dem Namen der Operation und (). Manche geben aber auch ein Objekt zurück, dadurch kann man gleich einen Operator davon Aufrufen oder man speichert sich es, mit einer variable zwischen. Außerdem haben sie manchmal Parameter, wie beim Initatiliesieren.
```

Comment:
Hast du nicht dafür den \q befehl eingeführt?
Der ganze Abschnitt ist außerdem noch sehr wirr und nutzt viele Begriffe die wir nicht benutzen wollen (Befehl, initializer, Zeiger, initialisieren).
Wenn du das nicht mehr in der PR fixen willst, dann mach wenigstens ein TODO dazu.

Reply:
habe gerade keine idee und deshalb ein Todo gemacht.